app/models/evidence_item.rb

```diff
@@ -74,6 +74,10 @@ def self.advanced_search_scope
     eager_load(:submitter, :disease, :source, :drugs, :open_changes, variant: [:gene, :variant_aliases])
   end
 
+  def self.variant_group_scope
+    eager_load(variant: [:variant_groups])
```

Comment:
I made this change but I'm still seeing a few queries when I hit this endpoint in the server (same with #353). Not sure if that is to be expected.